packages/aws-serverless/package.json

```diff
@@ -21,10 +21,6 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "import": {
-        "types": "./build/npm/types/index.d.ts",
-        "default": "./build/npm/esm/index.js"
-      },
```

Comment:
The alternative I guess would be to point to 

```json
"default": "./build/npm/cjs/index.js"
```

instead but I have no idea if this is better or would cause other issues 🤔 My gut feeling says issues because Node probably expects ESM when it resolves an `import` entry point (?)